[Marketplace Contribution] Cisco Umbrella Reporting - Content Pack Update (#35858)
demisto/content: Packs/CiscoUmbrellaReporting/ReleaseNotes/1_2_0.md

#### Integrations

##### Cisco Umbrella Reporting

> Added the categories parameter to support querying the API for specific category IDs in all commands (where supported)

Reviewer: ```suggestion
Added the `categories` parameter to support querying for specific category IDs in relevant commands.
```
